Library/Homebrew/diagnostic.rb

```diff
@@ -188,11 +188,14 @@ def __check_stray_files(dir, pattern, allow_list, message)
         files = Dir.chdir(dir) do
           (Dir.glob(pattern) - Dir.glob(allow_list))
             .select { |f| File.file?(f) && !File.symlink?(f) }
-            .map { |f| File.join(dir, f) }
+            .map do |f|
+              f.sub!(%r{/.*}, "/...") unless @verbose
+              File.join(dir, f)
+            end
         end
         return if files.empty?
 
-        inject_file_list(files.sort, message)
+        inject_file_list(files.sort.uniq, message)
```

Comment:
Might be nicer to pull the `.sort.uniq` above; it'll be a no-op if it's `empty?` anyway.

Reply:
OK, sure.